src/protocols/BUILD.gn

```diff
@@ -17,6 +17,8 @@ import("//build_overrides/chip.gni")
 static_library("protocols") {
   output_name = "libChipProtocols"
 
+  defines = ["CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONER_DISCOVERY_CLIENT=1"]
```

Comment:
Why do we need it? If it's needed to fix unit tests maybe we should enable it in conditional way. Now it will be also enabled for our platform and I don't think we want it.

Reply:
We need it due to changes regarding removing POSIX api, but right, it should be enabled in a conditional way. I will add it.